### PR TITLE
Added loading card and CSP

### PR DESCRIPTION
**Pull request by ChristopherJMiller**
- Added explicit CSP definitions to allow for cross-origin requests
- Added event loading card to ensure Card styling is exported (Bug related to the event cards being created post initial request, therefore leading to the initial production export not noticing its usage and not including the styles)

### CODE REVIEW

File: components/event.jsx [hunk 1]
```diff
@@ -14,10 +14,10 @@ const event = ({ name, description, datetime, location }) => (
           {description}
         </Card.Text>
         <FontAwesomeIcon className="mr-1" icon={faClock} fixedWidth />
-        {moment(datetime).format('ddd. MMMM Do, h:mma')}
+         {moment(datetime).format('ddd. MMMM Do, h:mma')}
```
Comment (ConnorJC3) [inline components/event.jsx]: Remove

File: components/event.jsx [hunk 2]
```diff
@@ -14,10 +14,10 @@ const event = ({ name, description, datetime, location }) => (
...
         <br />
         <FontAwesomeIcon className="ml-1 mr-1" icon={faMapPin} />
-        {location}
+         {location}
```
Comment (ConnorJC3) [inline components/event.jsx]: Remove

File: pages/index.jsx [hunk 3]
```diff
@@ -120,7 +121,7 @@ function Home() {
             <h2>Upcoming Events</h2>
             <hr />
             <Row className="justify-content-center">
-              {eventCards}
+              {eventCards ? eventCards : <Card><Card.Text>Loading...</Card.Text></Card>}
```
Comment (ConnorJC3) [inline pages/index.jsx]: Maybe should be `eventCards ?? <Card><Card.Text>Loading...</Card.Text></Card>`?

Comment (ConnorJC3) [reply pages/index.jsx]: (Or use `||` if you want to include falsey values for `eventCards`)

**Review by ConnorJC3 — CHANGES_REQUESTED** [review]

File: cloud-deploy.yaml [hunk 4]
```diff
@@ -63,6 +63,8 @@ metadata:
     kubernetes.io/ingress.class: "nginx"
     cert-manager.io/cluster-issuer: "letsencrypt-prod"
     nginx.ingress.kubernetes.io/from-to-www-redirect: "true"
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      more_set_headers "Content-Security-Policy: default-src 'none'; script-src 'self'; style-src 'self'; img-src 'self' data:; connect-src 'self' https://events.uf-ace.com; font-src 'self' data:; base-uri 'none'; form-action 'self'; frame-ancestors 'none'; navigate-to https:; block-all-mixed-content";
```
Comment (ConnorJC3) [inline cloud-deploy.yaml:67]: This doesn't seem to be based on our master CSP?

**Review by ConnorJC3 — CHANGES_REQUESTED** [review]

File: cloud-deploy.yaml (hunk 4 above)
Comment (ChristopherJMiller) [reply cloud-deploy.yaml:67]: This was based on the CSP used by the ace events front deployment. Do you have a link to the master CSP?

File: pages/index.jsx (hunk 3 above)
Comment (ChristopherJMiller) [reply pages/index.jsx]: Fixed in f8f64b1992724eb315c202799ff5ecab137011cd

File: components/event.jsx (hunk 2 above)
Comment (ChristopherJMiller) [reply components/event.jsx]: Fixed in f8f64b1992724eb315c202799ff5ecab137011cd

File: components/event.jsx (hunk 1 above)
Comment (ChristopherJMiller) [reply components/event.jsx]: Fixed in f8f64b1992724eb315c202799ff5ecab137011cd

**Review by ConnorJC3 — APPROVED** [review]
LGTM, **merge with squash**

**Pull request merged**